buildSrc/build.gradle

```diff
@@ -56,6 +65,12 @@ rocker {
     extendsModelClass "com.uber.okbuck.template.core.Rule"
 }
 
+tasks.withType(JavaCompile) {
+  if (!name.toLowerCase().contains("test")) {
```

Comment:
Why not run on tests?

Reply:
There are no tests

I guess it can be removed :)